src/Visitor/MutationsCollectorVisitor.php

```diff
@@ -58,19 +58,20 @@ public function __construct(array $mutators, string $filePath, array $fileAst, C
     public function leaveNode(Node $node)
     {
         foreach ($this->mutators as $mutator) {
-            if ($mutator->isFunctionBodyMutator() &&
-                !$node->getAttribute(WrappedFunctionInfoCollectorVisitor::IS_INSIDE_FUNCTION_KEY)) {
-                continue;
+            if (!$this->codeCoverageData->isLineFunctionSignature($this->filePath, $node->getLine())) {
```

Comment:
I'd say that we can save result into some variable as we're executing it at least 3 times